### ISSUE TITLE
Default values for settings changed

### ISSUE THREAD

**Issue by AR15senalOfCode** [BlafKing/spicetify-cat-jam-synced#8]
With your newest update, it seems like the default value for some settings have changed.

Specifically, "Method to calculate better BPM" should be, in my opinion, defaulted to "Track BPM", since it seems to be working much much better!

Thanks for this by the way ❤

**Comment by picklepopper1** [comment]
I agree with this. "Track BPM" should be the default setting until the other one is more accurate.

**Comment by Nuzair46** [comment]
ah sorry. i think i made a mistake with the dropdown.
will fix it.
